README.rst

```diff
@@ -6,24 +6,28 @@ cell_coadds
 |build|
 |coverage|
 
-.. |build| image:: https://github.com/lsst-dm/cell_coadds/actions/workflows/test.yaml/badge.svg?branch=main
-   :target: https://github.com/lsst-dm/cell_coadds/actions/workflows/test.yaml
+.. |build| image:: https://github.com/lsst/cell_coadds/actions/workflows/test.yaml/badge.svg?branch=main
+   :target: https://github.com/lsst/cell_coadds/actions/workflows/test.yaml
 
-.. |coverage| image:: https://codecov.io/github/lsst-dm/cell_coadds/branch/main/graph/badge.svg
-   :target: https://codecov.io/github/lsst-dm/cell_coadds
+.. |coverage| image:: https://codecov.io/github/lsst/cell_coadds/branch/main/graph/badge.svg
+   :target: https://codecov.io/github/lsst/cell_coadds
 
 
-``cell_coadds`` is a package being developed for eventual inclusion in the `LSST Science Pipelines <https://pipelines.lsst.io>`_.
+``cell_coadds`` is a package in the `LSST Science Pipelines <https://pipelines.lsst.io>`_.
 
-It will eventually include code for building coadds of astronomical images in small (few arcsecond) cells, in which only input images that fully contain a cell are included.
+It includes the data structures for defining coadds of astronomical images in small (few arcsecond) cells, in which only input images that fully contain a cell are included.
 This helps mitigate problems with PSF discontinuities that are present in traditional coadds.
 
-At present this package is a rapidly-evolving prototype, and it is being developed alongside the `LSST Dark Energy Science Collaboration's <https://lsstdesc.org/>`_ `descwl_coadd <https://github.com/LSSTDESC/descwl_coadd/>`_ package.
+To import this package, setup the package in your environment and then run:
+
+.. code-block:: python
+
+    >>> import lsst.cell_coadds
 
 Development workflow
 ====================
 
-This package currently relies exclusively on GitHub Actions for CI, which test it against the `stackvana <https://anaconda.org/conda-forge/stackvana>`_ conda-forge distribution of the latest weekly release of the LSST Science Pipelines.
+This package currently relies extensively on GitHub Actions for CI, which test it against the `stackvana <https://anaconda.org/conda-forge/stackvana>`_ conda-forge distribution of the latest weekly release of the LSST Science Pipelines.
```

Comment:
We're going to have to stop doing this, I think.  Once cell_coadds is in lsst_distrib it'll start being included in stackvana.

Reply:
Of course! I'd like to not remove it in this ticket though.